src/TokenRunner/DocBlock/MalformWorker/ParamNameTypoMalformWorker.php

```diff
@@ -37,25 +37,27 @@ public function work(string $docContent, Tokens $tokens, int $position): string
 
         $paramNames = $this->getParamNames($docContent);
 
+        $missArgumentNames = [];
         // remove correct params
         foreach ($argumentNames as $key => $argumentName) {
             if (in_array($argumentName, $paramNames, true)) {
                 $paramPosition = array_search($argumentName, $paramNames, true);
                 unset($paramNames[$paramPosition]);
-                unset($argumentNames[$key]);
+            } else {
+                $missArgumentNames[$key] = $argumentName;
```

Comment:
Make `$argumentNames` a list to hold all arguments and add the wrong argument to another argument instead of `unset`.